theme_yes/static/src/js/tour.js

```diff
@@ -8,41 +8,39 @@ var tour = require("web_tour.tour");
 
 const snippets = [
     {
-        id: 's_carousel',
-        name: 'Carousel',
+        id: 's_cover', // whatever the id, it works anyway
```

Comment:
yes, it is an id :/
Purpose is to allow tour to select this Cover later and not another one if your tour insert 2 'Cover'...
it could be the index 1,2,3,4...  it will works

name is to select the snippet to drop, id to find it e.g. in selectSnippetColumn, changeIcon, changeImage, ...

So, just be sure in case you drop 2 snippets identical to use coin and bird as id ;) :bird: